modules/lvms-monitoring-logical-volume-manager-operator.adoc

```diff
@@ -6,8 +6,7 @@
 [id="lvms-monitoring-using-lvms_{context}"]
 = Monitoring {lvms}
 
-When {lvms} is installed using the {product-title} Web Console, you can monitor the cluster by using the *Block and File* dashboard in the console by default.
-However, when you use {rh-rhacm} to install {lvms}, you need to configure {rh-rhacm} Observability to monitor all the {sno} clusters from one place.
+When you use {rh-rhacm} to install {lvms}, you need to configure {rh-rhacm} Observability to monitor all the {sno} clusters from one place.
```

Comment:
```suggestion
When you use {rh-rhacm} to install {lvms}, you must configure {rh-rhacm} Observability to monitor all the {sno} clusters from one place.
```
⚠️ ISG specifies "must" rather than "need to" for instances like this. @rohennes